x-pack/elastic-agent/pkg/agent/cmd/container.go

```diff
@@ -474,7 +474,7 @@ func kibanaClient(cfg kibanaConfig, headers map[string]string) (*kibana.Client,
 		IgnoreVersion: true,
 		Transport:     transport,
 		Headers:       headers,
-	}, 0)
```

Comment:
we should probably use `Elastic-Agent` or do you have a list, enum... other set of values you agreed on with anybody?